bot/start.ts

```diff
@@ -450,10 +453,12 @@ const initialize = (botToken: string, options: Partial<Telegraf.Options<MainCont
       if (!(await validateObjectId(ctx, orderId))) return;
       const order = await Order.findOne({ _id: orderId });
 
-      if (!order) return;
+      if (order === null) return;
 
       const buyer = await User.findOne({ _id: order.buyer_id });
       const seller = await User.findOne({ _id: order.seller_id });
+      if (buyer === null) return;
+      if (seller === null) return;
```

Comment:
@Mersho why are you changing behaviour here? converting from JS to TS should not change the behaviour of the program